Rocket.Chat/Managers/Socket/Response/SocketResponse.swift

```diff
@@ -10,41 +10,38 @@ import Foundation
 import SwiftyJSON
 import Starscream
 
-
 public struct SocketResponse {
     var socket: WebSocket?
     var result: JSON
 
     // JSON Data
+    // swiftlint:disable variable_name
     var id: String?
     var msg: ResponseMessage?
     var collection: String?
     var event: String?
-    
-    
+
     // MARK: Initializer
 
     init?(_ result: JSON, socket: WebSocket?) {
         self.result = result
         self.socket = socket
-        
         self.id = result["id"].string
         self.collection = result["collection"].string
-        
+
         if let eventName = result["fields"]["eventName"].string {
             self.event = eventName
         }
-        
+
         if let msg = result["msg"].string {
             self.msg = ResponseMessage(rawValue: msg) ?? ResponseMessage.Unknown
         }
     }
-    
-    
+
     // MARK: Checks
-    
+
     func isError() -> Bool {
-        if msg == .Error || result["error"] != nil {
+        if msg == .Error || result["error"] != JSON.null {
```

Comment:
I haven't tested yet, but I believe in this case, `nil` will not be same as `JSON.null`. Have you tried it?